src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/DataSetViewModels.cs

```diff
@@ -0,0 +1,87 @@
+#nullable enable
+using System;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Public.Data;
+
+/// <summary>
+/// Provides high-level information about a data set.
+/// </summary>
+public record DataSetViewModel
+{
+    /// <summary>
+    /// The ID of the data set. 
+    /// </summary>
+    public required Guid Id { get; init; }
+
+    /// <summary>
+    /// The title of the data set.
+    /// </summary>
+    public required string Title { get; init; }
+
+    /// <summary>
+    /// A summary of the data set’s contents.
+    /// </summary>
+    public required string Summary { get; init; }
+
+    /// <summary>
+    /// The status of the data set. Can be one of the following:
+    ///
+    /// - `Published` - the data set has been published and will receive updates
+    /// - `Deprecated` - the data set is being discontinued and will no receive updates
+    /// - `Withdrawn` - the data set has been withdrawn and can no longer be used
```

Comment:
This doesn't provide the full set of statuses. Would suggest that we remove them from here completely as they'll easily get out of sync (if we add any more statuses) and we don't need this level of detail as we don't really use the Swagger docs for the admin.